Data/market-hours/market-hours-database.json

```diff
@@ -37133,391 +37133,1840 @@
         "12/24/2020": "13:00:00"
       }
     },
-    "Index-usa-[*]": {
-      "dataTimeZone": "America/Chicago",
-      "exchangeTimeZone": "America/Chicago",
+	"Future-cds-[*]": {
```

Comment:
These changes should also be reverted I think? see https://github.com/QuantConnect/Lean/pull/5710